src/navigation/AddWalletNavigator.tsx

```diff
@@ -0,0 +1,59 @@
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
+import React, { useState } from 'react';
+import Routes from '@/navigation/routesNames';
+import { deviceUtils } from '@/utils';
+import { AddWalletSheet } from '@/screens/AddWalletSheet';
+import { ImportSeedPhraseSheet } from '@/screens/ImportSeedPhraseSheet';
+import { IS_ANDROID } from '@/env';
+import { SheetHandleFixedToTopHeight, SlackSheet } from '@/components/sheet';
+import { BackgroundProvider } from '@/design-system';
+import { StatusBar, View } from 'react-native';
+
+const Swipe = createMaterialTopTabNavigator();
+
+export const contentHeight =
+  deviceUtils.dimensions.height - SheetHandleFixedToTopHeight;
```

Comment:
Would be cool to use `useWindowDimensions` from react native which will change if the height changes
https://reactnative.dev/docs/usewindowdimensions